root/layout/header/bottom.tt

```diff
@@ -13,7 +13,7 @@
 
 <ul class="menu">
     <li class="about">
-        <span class="menu-header">[% l('About Us') %]</span>
+        <span class="menu-header">[% l('About Us') %]&nbsp;&#x25BE;</span>
```

Comment:
Can you add that to the react files too? But it'll need to look like `{'\xA0\u25BE'}` 'cause react makes it hard to insert arbitrary HTML.